lua/noice/lsp/signature.lua

```diff
@@ -205,14 +206,15 @@ function M:format_signature(sig_index, sig)
     self.message:newline()
   end
 
-  local count = self.message:height()
-  self.message:append(sig.label)
-  self.message:append(NoiceText.syntax(self.ft, self.message:height() - count))
   local param = self:active_parameter(sig_index)
-  if param then
-    self:format_active_parameter(sig, param)
+  for line in sig.label:gmatch("[^\r\n]+") do
+    self.message:append(line)
+    if param then
+      self:format_active_parameter(line, param)
```

Comment:
This will not have the desired effect in two cases:

1. `param` is NOT a `string`, but the `[i, j]` form, giving the offsets of the overall signature to highlight (some LSPs do this?)
2. `param` is a `string`, but there are multiple identical params (i.e. `_` in some languages?)

There's probably nothing that can be done about 2 and it was already a problem, except that instead of all identical parameters being highlighted it was just the first.